contentcuration/contentcuration/frontend/channelEdit/components/Clipboard/index.vue

```diff
@@ -20,19 +21,33 @@
           :flat="!elevated"
         >
           <VListTile class="grow">
-            <VListTileAction v-if="!refreshing && channels.length">
-              <Checkbox
-                ref="checkbox"
-                class="ma-0 pa-0"
-                :value="selected"
-                :label="selectionState ? '' : $tr('selectAll')"
-                :indeterminate="indeterminate"
-                @click.stop.prevent="goNextSelectionState"
-              />
-            </VListTileAction>
-            <VListTileContent>
+            <VSlideXTransition hide-on-leave>
```

Comment:
Noticing a little jankiness on the transition between the two. Might make sense to add `leave-absolute` to smooth out the switch between the clipboard and preview mode
![clipboard transition](https://user-images.githubusercontent.com/7447496/98426250-f54d0180-204c-11eb-8372-ded96957ac38.gif)